{
  "gqlStatus": "50N42",
  "gqlStatusDescription": "error: general processing exception - unexpected error. Failed to connect to server. Please ensure that your database is listening on the correct host and port and that you have compatible encryption settings both on Neo4j server and driver. Note that the default encryption setting has changed in Neo4j 4.0. Caused by: Failed to connect",
  "diagnosticRecord": {
    "OPERATION": "",
    "OPERATION_CODE": "0",
    "CURRENT_SCHEMA": "/"
  },
  "classification": "UNKNOWN",
  "name": "Neo4jError",
  "code": "ServiceUnavailable",
  "retriable": true,
  "originalLine": 78,
  "originalColumn": 26
}